ScarletEngine/include/Core/Engine.h

```diff
@@ -45,11 +44,11 @@ namespace ScarletEngine
 		Engine(Engine&&) = delete;
 		
 		/** List of tickables to update on a fixed-timestep */
```

Comment:
Why not std vector?

Reply:
Its not a list in the sense that its a list datastructure but rather just a "list" of objects. The underlying DS is a dynamic linear array (std::vector).